RecoTracker/IterativeTracking/python/DetachedQuadStep_cff.py

```diff
@@ -337,6 +360,11 @@
                                 detachedQuadStepTracks,
                                 detachedQuadStep)
 DetachedQuadStep = cms.Sequence(DetachedQuadStepTask)
+
+_DetachedQuadStepTask_trackingMkFit = DetachedQuadStepTask.copy()
+_DetachedQuadStepTask_trackingMkFit.add(detachedQuadStepTrackCandidatesMkFitSeeds, detachedQuadStepTrackCandidatesMkFit, detachedQuadStepTrackCandidatesMkFitConfig)
+trackingMkFitDetachedQuadStep.toReplaceWith(DetachedQuadStepTask, _DetachedQuadStepTask_trackingMkFit)
+
```

Comment:
we will need to test it on Phase2 as well .... ok, after the Run3 developments ;)

Reply:
Yes, but that will need more development than just adjusting configuration files.

Reply:
Phase2 adaptation is part of the plan, but will only start after we are done w Run3 preparation.